homeassistant/components/ihc/ihcdevice.py

```diff
@@ -90,3 +88,21 @@ def on_ihc_change(self, ihc_id, value):
         Derived classes must overwrite this to do device specific stuff.
         """
         raise NotImplementedError
+
+    @property
+    def device_info(self):
+        """Return the device info."""
```

Comment:
Adding device_info is great - but is it dependant on config-flow?

Reply:
I have removed the device_info. To be added in a later PR